curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e056bdde6ae6892ba2.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-58
 
 It is a bit boring for all the text to have the same `font-family`. You can still have the majority of the text `sans-serif` and make just the `h1` and `h2` elements different using a different selector.
 
-Style both the `h1` and the `h2` elements so that only these elements' text use `Impact` font.
+Style both the `h1` and the `h2` elements using a single selector so that these elements' text use Impact font.
```

Comment:
```suggestion
Style both the `h1` and the `h2` elements using a single selector so that these elements' text use `Impact` font.
```

We do want to leave the code tags around the font name, as we don't want it to be translated.

Reply:
Thank you. I completely forgot about it.